common/aggregate_functions/src/aggregator_common.rs

```diff
@@ -0,0 +1,53 @@
+use std::fmt::Display;
+
+use common_exception::ErrorCodes;
+use common_exception::Result;
+
+// Copyright 2020-2021 The Datafuse Authors.
```

Comment:
style.

Reply:
Auto import in vscode breaks this, maybe add Licence header checks.